### DISCUSSION TITLE
官网示例只有ts没有js  是否会增加js版本

### DISCUSSION THREAD

**Discussion by dxygit1** [element-plus/element-plus#16799] [Q&A]
![image](https://github.com/element-plus/element-plus/assets/57128304/aa74798d-d625-42ed-9e3b-638644bb89ff)
官网示例只有ts没有js  是否会增加js版本

**Comment by warmthsea** [comment]
> ![image](https://github.com/element-plus/element-plus/assets/57128304/aa74798d-d625-42ed-9e3b-638644bb89ff)
> 
> 官网示例只有ts没有js  是否会增加js版本

把类型删了就是 js
